docs/tutorials/PersistentServer.md

```diff
@@ -13,12 +13,10 @@ Cromwell remembers everything it knows!
 
 ### Let's get started!
 
-- Pull the MySQL docker image from dockerhub:
-`docker pull mysql:5.5`
```

Comment:
definitely don't need this?

Reply:
Not needed. `docker run` pulls images if they aren't local already